lock-api/src/test/java/com/palantir/lock/client/TransactionStarterTest.java

```diff
@@ -55,6 +57,7 @@
 
     private final StartTransactionsLockWatchEventCache lockWatchEventCache =
             spy(StartTransactionsLockWatchEventCache.createForTests());
+    private final ValueCacheUpdater valueCache = mock(ValueCacheUpdater.class);
```

Comment:
we do have a handy no-op implementation we could use, but this is fine, don't really care.